content/user/ezpages/ezpages_display.md

```diff
@@ -46,6 +46,12 @@ If you're in HTML (say in a list), you would use
 <li><a href="<?php echo zen_href_link(FILENAME_EZPAGES, 'id=19'); ?>"><?php echo NEW_PAGE_TITLE; ?></a></li>
 ```
 
+or if it's pure HTML (being edited in CKEditor for example), do to it without PHP would be 
```

Comment:
@scottcwilson Is thist a typo?  `do to it without PHP`

Reply:
Indeed it is!  Will fix on next checkin.